app/population/controllers/edit.test.js

```diff
@@ -1,11 +1,12 @@
 const proxyquire = require('proxyquire')
-const mockPopulationService = {}
+const mockPopulationService = {
+  create: sinon.stub(),
+  update: sinon.stub(),
+}
 
 const FormWizardController = require('../../../common/controllers/form-wizard')
 
-const Controller = proxyquire('./edit', {
-  '../../../common/services/population': mockPopulationService,
-})
+const Controller = proxyquire('./edit', {})
```

Comment:
It looks like we don't actually need to use proxyquire for this any more.

Reply:
Nice catch, updated!